### PR TITLE
Add Ubuntu install examples for using local mounted iso

### PR DESCRIPTION
**Pull request by keedya**
- Simply by downloading the ISO and mounting it to on-http static repo
 as follows:
 - mount -o loop /var/mirrors/ubuntu-14.04.4-server-amd64.iso on-http/static/http/ubuntu

### CODE REVIEW

**Comment by keedya** [comment]
@RackHD/corecommitters @zyoung51 @jlongever @tannoa2 @uppalk1

File: example/samples/install_ubuntu_payload_iso_full.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    "options": {
+        "defaults": {
+            "version": "trusty",
+            "repo": "http://172.31.128.1:9080/ubuntu",
+            "rootPassword": "RackHDRocks!",
+            "hostname": "rackhd-node",
+            "domain": "example.com",
+            "baseUrl": "install/netboot/ubuntu-installer/amd64",
+            "kargs": {
+                "live-installer/net-image": "http://172.31.128.1:9080/ubuntu/install/filesystem.squashfs"
```
Comment (benbp) [inline example/samples/install_ubuntu_payload_iso_full.json]: `http://{{ api.server }}/<rest of url` would be more flexible here.

File: example/samples/install_ubuntu_payload_iso_full.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
+{
+    "options": {
+        "defaults": {
+            "version": "trusty",
+            "repo": "http://172.31.128.1:9080/ubuntu",
```
Comment (benbp) [inline example/samples/install_ubuntu_payload_iso_full.json]: `http://{{ api.server }}/<rest of url\>`would be more flexible here.

**Comment by benbp** [comment]
👍

**Comment by keedya** [comment]
@benbp updated!!
Thanks!!

File: example/samples/install_ubuntu_payload_iso_minimal.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "options": {
+        "defaults": {
+            "repo": "http://{{ api.server }}/ubuntu",
```
Comment (yyscamper) [inline example/samples/install_ubuntu_payload_iso_minimal.json]: {{api.server}} has already contained `http://`, so you need remove it. see https://github.com/RackHD/on-tasks/blob/master/lib/task.js#L115

File: example/samples/install_ubuntu_payload_iso_minimal.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+            "version": "trusty",
+            "baseUrl": "install/netboot/ubuntu-installer/amd64",
+            "kargs": {
+                "live-installer/net-image": "http://{{ api.server }}/ubuntu/install/filesystem.squashfs"
```
Comment (yyscamper) [inline example/samples/install_ubuntu_payload_iso_minimal.json]: remove `http://`

**Comment by anhou** [comment]
:+1: 
@keedya I didn't pay attention for Ubuntu ISO installation, we want to do some following steps to make Ubuntu installation better and I've created a story to let iso installation parameters to be consistent with net installation.  the story link https://www.pivotaltracker.com/n/projects/1492892/stories/126851627   and also create a  story to analysis and validate repos , the story link https://www.pivotaltracker.com/story/show/126851439
Any problems, let me know.

**Comment by keedya** [comment]
@anhou sounds good!!

**Comment by keedya** [comment]
@yyscamper code updated!!
Thanks!

**Comment by yyscamper** [comment]
:+1:

**Pull request merged**